exercises/exercise_1/README.md

```diff
@@ -1,336 +1,197 @@
 
-Once deployed, a new link appears with the exposed port `8080`:
-![exposed port](open-port.png)
+For this exercise, we will start with an existing compose file. But, it has missing pieces in it, which we will fill in to learn about the various components of a compose file. We will then build on this file for the next exercises. Note that, when creating your own file in the future, only compose version 3 schema is supported in Docker App.
 
-* **Click on it**, it should open a new tab and display `Hello DockerCon`.
-* **Hit `Ctrl+C`** to stop and nicely remove the service.
-```sh
-^C
-Gracefully stopping... (press Ctrl+C again to force)
-Stopping workshop_hello_1_550ca1a27072 ... done
-```
+Many development teams have a `docker-compose.yml` file that is intended for development (mounts in source code, uses a dev-enabled container) and a `docker-stack.yml` file for production deploys (uses built images, no code mounting). Since Docker App requires a production deploy, it'll typically reflect the setup used in a `docker-stack.yml` file. For simplicity in this exercise (since we are using Docker Compose), we are using a `docker-compose.yml`.
 
-**NOTE:** that the containers are prefixed with the `current directory` then `service` then `replicas index`. The application name is the current directory.
 
-**NOTE:** Hit `Alt+Enter` to enter a *fullscreen* mode
+## The Application
 
-### Deploy using `docker stack`
+For this workshop, we will use a modified version of the [Voting App](https://github.com/dockersamples/example-voting-app). This application contains the following services:
 
-Another way to deploy your compose file is to use `docker stack deploy`. It introduces the concept of stack, which you can query directly from docker.
-During this workshop we will focus on `swarm` orchestrator, but the same commands work the same way targeting `kubernetes` orchestrator.
+- A Python webapp that provides the ability to vote between two options
+- A Redis cache to store new votes
+- A .NET Core worker to consume votes and store them in…
+- A Postgres DB backed by a Docker volume for storing results
+- A Node.js webapp to show the results of the voting in real time
 
-The `docker stack` commands
-```sh
-$ docker stack --help
+![Voting App Architecture](architecture.png)
 
-Usage:  docker stack [OPTIONS] COMMAND
 
-Manage Docker stacks
+## Finishing the Compose File
 
-Options:
-      --orchestrator string   Orchestrator to use (swarm|kubernetes|all)
+The compose file found in the `exercise_1` directory has all of the services needed for the application, but has a few missing pieces to help us learn a little bit more about Docker Compose. We will fill in a few pieces now, but will continue to evolve our compose file as we advance through the workshop.
```

Comment:
Put a direct link to the file? Hope [exercise1/docker-compose.yml](docker-compose.yml) works 😅